dominion-solutions/n8n
# Ethan sowell/cu camkkp/build a clockify writer for tags

# Details

Added ability to create tags in ClockifyWriter.

# Steps to test changes

1. Use npm run dev to start the localhost.
1. Add a Clockify regular node.
1. Select "Create" and "Tag" and put in all the information required, then click execute.
1. Make sure information returned is right and that a tag is made.
> Returns existing tag info if your tag name is a duplicate.

# Issues Addressed

[CU-camkkp](https://app.clickup.com/t/camkkp) - Build a Clockify Writer for Tags
